firebase-crashlytics/src/main/java/com/google/firebase/crashlytics/FirebaseCrashlytics.java

```diff
@@ -91,9 +92,12 @@ public class FirebaseCrashlytics {
     final ExecutorService crashHandlerExecutor =
         ExecutorUtils.buildSingleThreadExecutorService("Crashlytics Exception Handler");
 
-    CrashlyticsAppQualitySessionsSubscriber sessionsSubscriber =
-        new CrashlyticsAppQualitySessionsSubscriber(arbiter);
-    firebaseSessions.register(sessionsSubscriber);
+    // TODO(mrober): Can we determine at this point if the app includes the ndk?
+    // If we can set hasNdk to false for non-native apps, there will be less disk io.
+    CrashlyticsAppQualitySessionsStore appQualitySessionsStore =
+        new CrashlyticsAppQualitySessionsStore(fileStore, /* hasNdk= */ true);
```

Comment:
Where do we set this `hasNdk` to false for SDK app? realized this variable is declared as `final` so we ain't change the value during the app life cycle

Reply:
We removed the hasNdk boolean because we can never not persist on session, even if the app does not have the ndk, because it could still have anrs.